From 62bd3dffb7e2a6815c434a20848d25f050c53a38 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Sun, 24 Dec 2017 18:33:22 +0000
Subject: [linux-steam-integration 07/26] redirect: We might return NULL here.

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 src/redirect/main.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/redirect/main.c b/src/redirect/main.c
index 1b6c142..50d8d45 100644
--- a/src/redirect/main.c
+++ b/src/redirect/main.c
@@ -363,6 +363,10 @@ _nica_public_ struct passwd *getpwuid(uid_t uid)
          * let us override the home directory to be correct.
          */
         ret = lsi_table.getpwuid(uid);
+        if (!ret) {
+                return NULL;
+        }
+
         if (uid == getuid() && snap_root && *snap_root) {
                 ret->pw_dir = (char *)snap_root;
         }
-- 
2.20.0

