From a951a4991033f428c5ba710f687a4142469b40ae Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey.doherty@intel.com>
Date: Fri, 21 Dec 2018 14:22:48 +0000
Subject: [linux-steam-integration 14/26] frontend: Apply update-format.sh

Signed-off-by: Ikey Doherty <ikey.doherty@intel.com>
---
 src/frontend/main-window.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/frontend/main-window.c b/src/frontend/main-window.c
index ad993ce..75719f0 100644
--- a/src/frontend/main-window.c
+++ b/src/frontend/main-window.c
@@ -114,7 +114,7 @@ static void lsi_settings_window_init(LsiSettingsWindow *self)
                 lsi_config_load_defaults(&self->config);
         }
 
-                /* Conditionally apply CSD */
+        /* Conditionally apply CSD */
 #if GTK_CHECK_VERSION(3, 12, 0)
         xdg_desktop = g_getenv("XDG_CURRENT_DESKTOP");
         if (xdg_desktop && (strstr(xdg_desktop, "GNOME:") || strstr(xdg_desktop, ":GNOME") ||
-- 
2.20.0

