From 9624bde1b9101bf7fa7c9b7d40d1d2700a508df8 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey.doherty@intel.com>
Date: Tue, 1 Jan 2019 14:04:34 +0000
Subject: [linux-steam-integration 15/31] mkgobuildy: Gracefully handle missing
 linux32 availability

Signed-off-by: Ikey Doherty <ikey.doherty@intel.com>
---
 mkgobuildy.sh | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/mkgobuildy.sh b/mkgobuildy.sh
index efdff2f..ffe19f0 100755
--- a/mkgobuildy.sh
+++ b/mkgobuildy.sh
@@ -2,7 +2,8 @@
 set -e
 
 if [[ ! -d buildroot/build32 ]]; then
-    CC="gcc -m32" linux32 meson buildroot/build32 -Dwith-libressl-mode=shim --prefix=/usr --sysconfdir=/etc --datadir=/usr/share --libdir=/usr/lib32 -Dwith-shim=none -Dwith-new-libcxx-abi=true --buildtype debugoptimized
+    ASFLAGS="$ASFLAGS --32" CFLAGS="$CFLAGS -m32" CXXFLAGS="$CXXFLAGS -m32" LDFLAGS="$LDFLAGS -m32" \
+    meson buildroot/build32 -Dwith-libressl-mode=shim --prefix=/usr --sysconfdir=/etc --datadir=/usr/share --libdir=/usr/lib32 -Dwith-shim=none -Dwith-new-libcxx-abi=true --buildtype debugoptimized
 fi
 
 if [[ ! -d buildroot/build64 ]]; then
-- 
2.20.0

