From 8d9aac19f631d4bf7924da34b3c0bc4cf93520cf Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey.doherty@intel.com>
Date: Mon, 7 Jan 2019 11:39:19 +0000
Subject: [linux-steam-integration 26/26] TEST_LIST: Document fact that Don't
 Starve works now

Signed-off-by: Ikey Doherty <ikey.doherty@intel.com>
---
 TEST_LIST | 1 +
 1 file changed, 1 insertion(+)

diff --git a/TEST_LIST b/TEST_LIST
index b2a64c3..7c91b61 100644
--- a/TEST_LIST
+++ b/TEST_LIST
@@ -6,6 +6,7 @@ Initial test list for next release of LSI
         - Vendored SDL is patched in non upstreamable fashion
           https://forums.kleientertainment.com/forums/topic/30218-crash-on-linux-steam-without-error/?page=0&tab=comments#comment-386249
           We'll need to allow certain games to vendor SDL now.
+          Fixed in https://github.com/clearlinux/linux-steam-integration/commit/c4ded161159d2d6faa5a1dba45af8426738632b7
  - Cities:Skylines
  - Various Source titles (L4D2, Portal, etc)
  - Various Feral titles (2018/2019 kits)
-- 
2.20.0

